AtbIntegrationTests/AtbIntegrationTests.swift

```diff
@@ -276,6 +275,11 @@ class Springboard {
             let springboardFrame = springboard.frame
             icon.press(forDuration: 1.3)
             
+            let rearrangeButton = springboard.buttons["Rearrange Apps"]
```

Comment:
Strange this new "Rearrange Apps" button from Apple!

Reply:
Yes! Seems they struggled with force touch and just decided to change it.. but it's weird. :)